experiments/can-we-drive-traffic-from-wdtk-to-asktheeu.md

```diff
@@ -0,0 +1,30 @@
+# Can we drive traffic from WDTK to AskTheEU?
+
+We wanted to know if adding adverts with info about and a link to AsktheEU.org to WhatDoTheyKnow would drive traffic to AsktheEU. E.g. Are WDTK users interested in requesting information from EU institutions?
+
+- [issue#50](https://github.com/mysociety/alaveteli-experiments/issues/50)
+
+## Answer
+
+**TL;DR:** Yes.
+
+![AsktheEU advert](https://cloud.githubusercontent.com/assets/739624/15118227/319e97ec-1603-11e6-9c11-56c3bcad423d.png)
```

Comment:
I'd originally intended that we'd store the screenshot in our repo rather than relying on GitHub's own cache but this is fine. Next time we may subject you to the "joy" of file uploads 😈